test/functional/test_site/expected/markbind/css/markbind.css

```diff
@@ -124,7 +124,7 @@ code.hljs:hover {
 header.header-fixed {
     position: fixed;
     width: 100%;
-    z-index: 9999;
+    z-index: 1000;
```

Comment:
Hmm this diff doesn't seem to be related to the changes in this PR. 

Seems to be related to https://github.com/MarkBind/markbind/pull/1070. Let's fix this in another PR? Also, it seems like CSS diffs aren't being checked by our tests 😅

Reply:
125ac543da90753aa13f8da9a7caaffa34cf617a, was indeed #1070's test file updates

> Seems to be related to #1070. Let's fix this in another PR? Also, it seems like CSS diffs aren't being checked by our tests 😅

I checked the test script briefly, it seems that only `siteData.json` and `.html` files are diffed. 
The likely original reason was to avoid diffing large library files ( eg `bootstrap.min.css` ); Perhaps a better solution would be to exclude these files explicitly. Will open as an issue!